Fixed a bug with sending mail (#11495)
demisto/content: Packs/Active_Directory_Query/ReleaseNotes/1_1_5.md

#### Scripts
##### IAMInitADUser

> - Fixed a bug where email was sent even though the create user flow failed.

Reviewer: ```suggestion
- Fixed an issue where an email was sent even though the create user flow had failed.
```
